vendor/engines/split_accounts/app/services/split_accounts/journal_row_converter.rb

```diff
@@ -0,0 +1,43 @@
+module SplitAccounts
+
+  # TODO: use an instance rather than a class service object and store journal
+  # and possibly order_detail
+  class JournalRowConverter < ::JournalRowConverter
```

Comment:
Seems like we'd end up with a lot of duplicate logic between ` SplitAccounts::JournalRowConverter` and `Nu::SplitAccounts::JournalRowConvert`. Maybe I'm not quite understanding what you're planning.